### PR TITLE
writeup implemented, cleanup code, added few more graphs

### PR DESCRIPTION
**Pull request by parminpatel28**

### CODE REVIEW

**Review by eevanwong — APPROVED** [review]
lgtm

File: pages/profile/academics/index.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 import Segment from "../../../components/layout/segment/segment";
+
+//graphing
 import DoughnutChart from "../../../components/charts/doughnut";
 import Scatterplot from "../../../components/charts/scatterplot";
-import BarGraph from "../../../components/charts/bar";
+import Bar from "../../../components/charts/bar";
+
+//writeup
+import Text from "../../../text1A";
```
Comment (shivam-sh) [inline pages/profile/academics/index.js:13]: We’re moving the write ups to firebase, we can remove the text1A file & references

File: pages/profile/academics/index.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
 import Head from "next/head";
 import React from "react";
+//styles
```
Comment (shivam-sh) [inline pages/profile/academics/index.js:3]: Nitpick but can you add a new line above the comment

**Review by ebrahim02-ag — APPROVED** [review]
good good

File: pages/profile/co-op/index.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
 import Head from "next/head";
+//Styles
```
Comment (shivam-sh) [inline pages/profile/co-op/index.js:2]: The same reviews as above but in all the files

File: text1A.js [hunk 4]
```diff
@@ -1,17 +1,81 @@
 module.exports = {
   text: {
```
Comment (shivam-sh) [inline text1A.js:2]: Like I said, this is moving to the firebase, we can talk about how to structure this in the meeting on Saturday if you want but having this in the codebase doesn’t make sense for now

**Review by shivam-sh — CHANGES_REQUESTED** [review]
Left some comments

**Review by shivam-sh — APPROVED** [review]

**Pull request merged**